pkgs/development/libraries/ndi/default.nix

```diff
@@ -35,20 +40,20 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir $out
-    mv bin/x86_64-linux-gnu $out/bin
+    mv bin/${ndiPlatform} $out/bin
     for i in $out/bin/*; do
+      if [ -L "$i" ]; then continue; fi
       patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" "$i"
     done
     patchelf --set-rpath "${avahi}/lib:${stdenv.cc.libc}/lib" $out/bin/ndi-record
-    mv lib/x86_64-linux-gnu $out/lib
+    mv lib/${ndiPlatform} $out/lib
```

Comment:
Can't we make our life simple here and use a wildcard?

Reply:
No, otherwise you'd ship binaries for foreign architectures that wouldn't be used.